src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -122,11 +122,15 @@ Statement | Action
 
 ### Cluster versions and upgrades
 
-The standby cluster must be at the same version as, or one version ahead of, the primary's application virtual cluster at the time of cutover.
+When PCR is enabled, we recommend upgrading with the following procedure. This upgrades the standby cluster before the primary cluster. It is preferable to avoid a situation where the virtual cluster, which is being replicated, is a version higher than what the standby cluster can serve if you were to cut over. Within the primary and standby CockroachDB clusters, the system virtual cluster must be at a cluster version greater than or equal to the virtual cluster:
```

Comment:
"It is preferable to avoid a situation where the virtual cluster, which is being replicated, is a version higher than what the standby cluster can serve if you were to cut over." not sure if we need  this sentence